transportable-udfs-trino-plugin/src/main/java/com/linkedin/transport/trino/TransportConnector.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.trino;
+
+import io.trino.spi.connector.Connector;
+import io.trino.spi.connector.ConnectorMetadata;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.connector.ConnectorTransactionHandle;
+import io.trino.spi.function.FunctionId;
+import io.trino.spi.function.FunctionProvider;
+import io.trino.spi.transaction.IsolationLevel;
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.Objects.*;
+
+
+public class TransportConnector implements Connector {
```

Comment:
Could you add java doc for these new classes?

Reply:
Added